acceptance_tests/features/steps/enrolment_status_disable.py

```diff
@@ -9,6 +11,27 @@ def internal_user_view_ru_details(_):
     reporting_unit.go_to('49900000001')
 
 
+@when('the user clicks either disable or re-enable')
+def click_enrolment_status_link(_):
+    browser.find_by_id('change-enrolment-status').click()
+
+
+@when('the user clicks the confirm button')
+def click_confirm_button(_):
+    browser.find_by_id('confirm-btn').click()
+
+
+@then('the user is redirected to a confirmation screen')
+def confirm_verification_screen(_):
+    assert 'Edit collection exercise details' in browser.title
+
+
+@then('the user is redirected to the respondents page and a success message is displayed')
+def confirm_success_message(_):
+    assert 'Respondents | Survey Data Collection' in browser.title
+    assert 'Enrolment status changed' in browser.find_by_id('success').text
```

Comment:
Are both these asserts necessary to confirm the step has been met?  Where possible, a test should assert a single thing - but that's not always possible.

Reply:
I think so, part of the test is making sure it doesn't redirect to the RU tab- and if it does the other assert would return true also

Reply:
Fair play, consider this resolved